### ISSUE TITLE
Emails for sending account information

### ISSUE THREAD

**Issue by hao-fang** [acl-org/acl-2020-virtual-conference#48]
**Number of Volunteers**: 1

~We need to send the password for both the website account & the RocketChat account.
It would be better if we can use the same password / common login. But this depends on #47.~ (this is done)

- [ ] Email template (need to send a draft to Dan & PCs)
  * https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pool-settings-message-templates.html
- [ ] Configure from & to email address (need to check with Dan & PCs about the email addresses; may need to check with Nitin to get the SES permission)
![image](https://user-images.githubusercontent.com/9957482/84538781-116b9300-aca7-11ea-92ae-610702f1f133.png)

### Emails sending to attendees
- [ ] Instruction to update displayed name (with Pronoun?)

### Additional email sending to authors
- [x] Zoom account(s)
- [x] RocketChat channel(s)
- [x] (optional) paper links
- [x] Q: Should we send emails to registered authors first?

**Comment by hao-fang** [comment]
Current message template. Need to edit this.

Subject: `Your temporary password for ACL2020`
Body:
```
Welcome to ACL2020! <br>
Please find your account information below. <br>
Username: {username} <br>
Password: {####} <br>
<br>
1. Please go to https://acl2020.rocket.chat/, and click the “Login with ACL2020 account (beta)“. <br>
2. On the login page, enter your username (email) & password. You will be prompted to change your password. You can use the same one if you want. <br>
3. Then you will be prompted to register your username for RocketChat. This should NOT be your email address. <br>
<br>
-- ACL2020 Organising Committees
```

**Comment by hao-fang** [comment]
@vladdy I changed this issue to be "on hold" because we still need to figure out a few details before deciding how to proceed.
I'll let you know when we can start working on this.

**Comment by hao-fang** [comment]
Current template as of June 24.

```
Welcome to ACL 2020! <br>
Please find your account information below. <br>
Username: {username} <br>
Password: {####} <br>
<br>
1. Please go to https://virtual.acl2020.org/, and login with your username and password. You will be prompted to change your password. You can use the same one if you want. <br>
2. Once you are logged in, please go to the "Chat" tab, and click the “Login with ACL2020 account“ button to set up your RocketChat account. You will be prompted to register your RocketChat username. The username should NOT be your email address. <br>
<br>
-- ACL2020 Organising Committees
```

**Comment by hao-fang** [comment]
From ICLR

```
Welcome to ICLR 2020, and in its fully virtual format. This email has information to help you prepare and fully-participate in the conference. We look forward to meeting you at a poster or social next week!

Conference Portal
You must first log into your account at iclr.cc. Once logged in, you will see your name in the top-right corner of the page.  If you have trouble logging in, try resetting your password, if that doesn’t work, email Brad Brockmeyer <brockmeyer@salk.edu> for assistance;  mention your iclr.cc email address. 

Once logged in, you can access the conference portal at https://iclr.cc/virtual/. The site navigation and components should be intuitive. Its key components are:

Schedule: https://iclr.cc/virtual/calendar.html this page lists all the live sessions of the conference.

Papers: https://iclr.cc/virtual/papers.html?filter=keywords this page allows you search for paper by keyword, author, title, or to simply browse all the accepted papers.

Help: https://iclr.cc/virtual/about.html this is where you will find the answer to any question you may have during the conference and particularly during your own presentation

Planning your Time
We recommend exploring the site in advance, and exploring the papers and talks you want dive deeper into. 

Download the calendars for the workshops, main conference and socials. You can use a calendar tool to plan your time. 

Every paper has a 5min video associated with it, so you can get an overview of the paper. Each paper page has a chat room associated with it, so you can discuss directly with the authors and other participants. You watch these ant any time that is convenient for you.

Each day there are 5 poster sessions at time zones across the world. There are also live Q&A settings with the authors. You want to plan your time to join these sessions directly.

You can easily bookmark page you want to return to, join channels for all the papers you are interested in and use your Rocket.Chat to note all the papers you are 

Rocket.Chat
We will use Rocket.Chat for asynchronous interaction between authors and participants. You were automatically registered in the ICLR Rocket.Chat using your email address (at the address you received this email). Please go to https://iclr.rocket.chat/ to reset your password and join the other participants. 

You can access the ICLR Rocket.Chat by either typing https://iclr.rocket.chat/ in your browser’s address bar or by going to Chat in the ICLR Web Portal (https://iclr.cc/virtual/chat.html). There is also a Rocket.Chat app you can use. There are several channels available, and two that will be useful are #announcements and #helpdesk.

Zoom
We will use Zoom extensively for all the conference’s live interaction. You will find links for Live sessions across the site. Some things you can do in advance.

Update your zoom client to the latest version.

For security reasons, you may not be permitted/able to use the zoom app. You can still participate by using zoom in the browser. See the FAQ for more details. 

See the FAQ for other security aspects we have put in place. 

General
All participants are expected to adhere to the code of conduct. To report a violation of the Code, the code of conduct has a section on reporting. Again, see FAQ on the portal.

For help navigating the site, finding papers and other aspects of the portal, please see the FAQ or use the #helpdesk. There are volunteers on-call who are able to help.

If you have issues logging into the site, with registration, or issues not covered in the FAQ, contact us at iclr2020-virtual -at- googlegroups.com

Stay healthy, and we look forward to having you in our virtual ICLR2020.

-- ICLR2020 Organising Committees
```
